apidoc/Titanium/UI/Window.yml

```diff
@@ -784,6 +784,9 @@ properties:
     type: Boolean
     since: 3.6.0
     availability: creation
+    deprecated:
+        since: 6.2.0
+        notes: Deprecated in AppCompat theme. The same behaviour can be achived by using Toolbar.
```

Comment:
So it still work? When you deprecate it, it still has to work. In case it does not anymore, it has to go into a major version (7.0.0).